saleor/webhook/payloads.py

```diff
@@ -535,6 +537,42 @@ def generate_list_gateways_payload(
     return json.dumps(payload)
 
 
+def _generate_payload_for_shipping_method(method: ShippingMethod):
+    payload = asdict(method)
+    payload["id"] = graphene.Node.to_global_id("ShippingMethod", payload["id"])
+    return payload
+
+
+def generate_excluded_shipping_methods_for_order_payload(
+    order: "Order",
+    available_shipping_methods: List[ShippingMethod],
+):
+    order_data = json.loads(generate_order_payload(order))[0]
+    payload = {
+        "order": order_data,
+        "shipping_methods": [
+            _generate_payload_for_shipping_method(shipping_method)
+            for shipping_method in available_shipping_methods
+        ],
+    }
+    return json.dumps(payload, cls=CustomJsonEncoder)
+
+
+def generate_excluded_shipping_methods_for_checkout_payload(
+    checkout: "Checkout",
+    available_shipping_methods: List[ShippingMethod],
+):
+    checkout_data = json.loads(generate_checkout_payload(checkout))[0]
```

Comment:
@kczan 
Are you sure that all required data are prefetched?
We should add a benchmark for this case(Remember to use multiline order).